extensions/wikia/AdEngine/js/provider/factoryWikiaGpt.js

```diff
@@ -24,6 +35,10 @@ define('ext.wikia.adEngine.provider.factory.wikiaGpt', [
 	function createProvider(logGroup, providerName, src, slotMap, extra) {
 		extra = extra || {};
 
+		if (extra.overrideSizesPerCountry && extra.overrideSizesPerCountry[country]) {
+			overrideSizes(slotMap, extra.overrideSizesPerCountry[country]);
```

Comment:
You're totally right - `helper` doesn't tell anything. And what about `overrideSizes`? I thought in this way:
FactoryWikiaGpt creates provider with specific slot map (which defines list of supported slots). It also prepares `canHandleSlot` method (it would be a bit harder to override in helper). It also has only one additional dependency (geo). I just reordered dependencies in alphabetical way. GPT helper is the last point of our integration between AdEngine and googletag. That's why decided to place it in factory.

Maybe it's worth to call for another pair of eyes :D @jacekjursza what do you think?